### DISCUSSION TITLE
Bugs in the Github UI

### DISCUSSION THREAD

**Discussion by travisdowns** [community/community#26802] [Actions]
<p>Where should bugs in the github UI be reported? There doesn’t seem to be a public repository for it on github.</p>

**Comment by Zerotask** [comment, marked as answer]
<p>There is indeed no uniform platform to communicate bugs.<br>
From my experience, you can use a form (like this: <a href="https://support.github.com/contact/feedback" class="inline-onebox" rel="noopener nofollow ugc">Share feedback - GitHub Support</a>), use this forum here or the discussion plattform on their feedback repository <a href="https://github.com/github/feedback/discussions" class="inline-onebox" rel="noopener nofollow ugc">Discussions · github/feedback · GitHub</a></p>

**Comment by otter-computer** [comment]
<p>Echoing what <a class="mention" href="/zerotask">@Zerotask</a> says, the <a href="https://support.github.com/contact/feedback">feedback contact form</a> is probably the best place to report UI issues and the support team will make sure it gets in front of the right team :sparkles:</p>

**Comment by travisdowns** [comment]
<p>Thanks <a class="mention" href="/zerotask">@Zerotask</a> and <a class="mention" href="/thomasshaped">@thomasshaped</a>. I’m a bit reluctant to use a feedback form since in my experience those are generally fire-and-forget things where you launch your feedback into the ether and never hear anything.</p>
<p>Something like a GitHub issue is much nicer since you get feedback, can track its progress, etc.</p>
<p>I forgot to mention that this is with the Github Actions UI specifically: is the answer the same in that case?</p>
